package.json

```diff
@@ -116,6 +116,7 @@
     "webpack-dev-server": "2.1.0-beta.9"
   },
   "scripts": {
+    "postinstall": "make install-selenium",
```

Comment:
I'd like to be sure selenium is correctly installed each time I do an `npm install`. Would it be ok to add a condition checking for `development` env in the makefile ?